src/pages/lobby/[id]/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { useRouter } from "next/router";
+import useStore from "../../../ions/useStore";
+
+const Page = () => {
+	const name = useStore(state => state.name);
+	const {
+		query: { id },
+	} = useRouter();
+
+	return (
+		<>
+			<h1>Lobby: {id}</h1>
+			<h2>Hello {name}</h2>
+		</>
+	);
+};
+
+export default Page;
```

Comment:
would always prefer named exports. here are a few cons of it:
https://www.bundleapps.io/blog/use-named-exports-over-default-exports-in-javascript

Reply:
since this is next.js routing, you probably wont ever export something else than the page, in general you want your components as small as possible, so i dont see a drawback in default exports, if there is only one export from that file and there only should be one for my understanding.

Reply:
Thanks a lot for the article John! Will definitely use named exports in the future. :D

Reply:
have to correct myself, a drawback is definitely the Explicit over implicit thing! I got that :)